### DISCUSSION TITLE
Stop PBFLASH drive on mac??

### DISCUSSION THREAD

**Discussion by beetlegigg** [micropython/micropython#14325] [STM32 / Pyboard] [answered]
I was wondering if there is a way to stop a drive named PYBFLASH being created whenever I connect my pyboardv1.1 to my old mac computer.  Why? well its because I usually forget to eject the drive before unplugging the board from the computers usb port.  I upload the files to the pyboard with Thonny or even mpremote and dont really need or want the pybflash drive.  

My old mac is not very forgiving if I do this a few times and something goes wrong with other stuff connected to the macs' usb ports like my mouse pointer will becomes erratic and jerky in its movements.  The only resolution to this mac wobbly that I've found is to reboot the mac. 
I would note that the same thing often happens if I reload a new micropython firmware to a board like the rpi pico when, at the end of the process, the RPI drive gets shutdown but not first ejected by the install process.  Often this means I have to reboot the old mac.   As upgrading new firmware does not take place too often this is not really a problem, but I was hoping there is a way to get mp on the pyboard working like it does on the rpi pico and it can be made to stop creating this drive when I'm just programming the pyboard.

**Comment by scruss** [comment, marked as answer]
change `pyb.usb_mode('VCP+MSC')` in boot.py to `pyb.usb_mode('VCP')` and reboot. It will go away.
